src/app/components/SwapStatusCard.jsx

```diff
@@ -20,32 +20,45 @@ const StatusFooter = ({ className, children, ...props }) => (
   </CardFooter>
 )
 
+const priceChange = (timestamp, asset) => {
+  const { change1, change7d, change24 } = asset
+  const daysSinceTrade = (Date.now() - timestamp) / 86400000
+  const timespan = daysSinceTrade < 1 ? 'Last 24hr: ' : daysSinceTrade >= 2 ? 'Last 7d: ' : 'Last 1d: '
+  var priceChangeText = daysSinceTrade < 1 ? change1 : daysSinceTrade >= 2 ? change7d : change24
+  return (
+    <span>{timespan}<ChangePercent>{priceChangeText}</ChangePercent><ArrowIcon style={arrowStyle} size={.58} dir={priceChangeText < 0 ? 'down' : 'up'} color={priceChangeText < 0 ? 'danger' : 'success'}/></span>
+  )
+}
+
 export default compose(
   setDisplayName('SwapStatusCard'),
   setPropTypes({
     swap: PropTypes.object.isRequired,
     showWalletLabels: PropTypes.bool,
     showFees: PropTypes.bool,
     showDetails: PropTypes.bool,
+    isExpanded: PropTypes.bool
```

Comment:
This prop name collides with the `isExpanded` state which can be confusing. I would rename the prop to `expanded` and the state to `isExpandedState` to differentiate the two. Treat the prop as a manual override for the state. If `expanded` is provided, the card will always be opened/closed and cannot be toggled. If expanded is not specified (use default value `null`), then the card can be toggled and `isExpandedState` controls the whether it's opened/closed.

Consider going about it like so:
```
withProps({
  expanded: PropTypes.boolean,
}),
defaultProps({
  expanded: null,
}),
withToggle('expandedState'),
withProps(({ expanded, isExpandedState, toggleExpandedState }) => ({
  isExpanded: expanded === null ? isExpandedState : expanded,
  togglerProps: expanded === null ? { tag: Button, color: 'ultra-dark', onClick: toggleExpandedState } : {}
})
...
<CardBody className='...' style={...} {...togglerProps}>
```

Reply:
works perfect! so withToggle handles the expandedState piece if nothing is passed in otherwise it defaults to the expanded prop.